StructureDefinition/Extension-England-COVID19TestResult.xml

```diff
@@ -58,41 +42,45 @@
       </slicing>
       <min value="0" />
     </element>
-    <element id="Extension.extension:primaryRole">
+    <element id="Extension.extension:reportOrigin">
       <path value="Extension.extension" />
-      <sliceName value="primaryRole" />
+      <sliceName value="reportOrigin" />
+      <short value="Indicates the source of a secondarily reported record" />
+      <definition value="Source of record creation - i.e. the digital service used to record the test registration.&#xD;&#xA;&#xD;&#xA;The source of the data when the report of the observation event is not based on information from the person who performed the observation." />
       <min value="0" />
       <max value="1" />
     </element>
-    <element id="Extension.extension:primaryRole.url">
+    <element id="Extension.extension:reportOrigin.url">
       <path value="Extension.extension.url" />
-      <fixedUri value="primaryRole" />
+      <fixedUri value="reportOrigin" />
     </element>
-    <element id="Extension.extension:primaryRole.value[x]">
+    <element id="Extension.extension:reportOrigin.value[x]">
       <path value="Extension.extension.value[x]" />
       <type>
-        <code value="boolean" />
+        <code value="CodeableConcept" />
       </type>
     </element>
-    <element id="Extension.extension:status">
+    <element id="Extension.extension:administrationMethod">
       <path value="Extension.extension" />
-      <sliceName value="status" />
+      <sliceName value="administrationMethod" />
+      <short value="Used to convey how the test was administered" />
+      <definition value="used to convey how the test was administered. Possible values are &quot;self&quot; or &quot;health_care_professional&quot;" />
       <min value="0" />
```

Comment:
This minimum cardinality of 0 was not part of the original extension you worked on. 
[ExtensionCOVID19TestResult](https://simplifier.net/nhsdigital/68172784-12f2-4437-8680-2595272a885f)